fulp_modules/features/antagonists/bloodsuckers/code/powers/_powers.dm

```diff
@@ -139,12 +139,15 @@
 				return FALSE
 	return TRUE
 
-/// NOTE: With this formula, you'll hit half cooldown at level 8 for that power.
 /datum/action/cooldown/bloodsucker/StartCooldown()
-	// Calculate Cooldown (by power's level)
-	if(power_flags & BP_AM_STATIC_COOLDOWN)
+	// Check for power flags first
+	if(!power_flags)
+		return
+	if(BP_AM_STATIC_COOLDOWN)
 		cooldown_time = initial(cooldown_time)
-	else
+	if(!BP_AM_VERY_DYNAMIC_COOLDOWN) // If this is TRUE then 'cooldown_time' doesn't need to be altered.
```

Comment:
This still needs to be compared to power_flags

Reply:
_See the reply to the requested change immediately above this one._